src/tools/pub/cmd/pub-global.md

```diff
@@ -137,9 +136,9 @@ may vary for different versions of Windows.
 You can now directly invoke the command:
 
 {% prettify none tag=pre+code %}
-$ mkdir angular_project
-$ cd angular_project
-$ [!stagehand web-angular!]
+$ mkdir web_project
```

Comment:
This made sense for stagehand, but let's assume a prepopulated app for webdev.

```suggestion
```

Reply:
Good point!